project/plugins.sbt

```diff
@@ -3,4 +3,5 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
   Resolver.ivyStylePatterns
 )
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.24.0")
+addSbtPlugin("uk.gov.hmrc"   % "sbt-auto-build" % "3.24.0")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt"   % "2.5.4")
```

Comment:
intellij auto formatted when I made changes, so added this so that we can easily fmt everything / check everything is fmt'ed in jenkins